crates/driver/src/domain/competition/mod.rs

```diff
@@ -386,10 +386,31 @@ pub struct Solved {
     pub gas: Option<eth::Gas>,
 }
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub struct Amounts {
-    pub sell: eth::TokenAmount,
-    pub buy: eth::TokenAmount,
+    pub side: order::Side,
+    pub sell: eth::Asset,
+    pub buy: eth::Asset,
+    pub traded_sell: eth::TokenAmount,
+    pub traded_buy: eth::TokenAmount,
+}
+
+impl Default for Amounts {
```

Comment:
Is this used a lot? Would be great if we could remove the `Default` implementation if possible.

Reply:
removed